packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadOptions/ReactionButton/ReactionButton.tsx

```diff
@@ -143,7 +143,7 @@ export const ReactionButton = ({
         >
           <CWUpvote
             onClick={handleVoteClick}
-            voteCount={reactors.length}
+            voteCount={thread.reactionWeightsSum || reactors.length}
```

Comment:
if you put `thread.reactionWeightsSum || reactors.length` here, shouldn't it be the same in other voteCount `props` above in this file?

Reply:
Yes! Updated!